frontline-functions-quickstart/.env.example

```diff
@@ -0,0 +1,45 @@
+# required: true
+# format: sid
+# description: Your Twilio Account SID. Can be found in the Console.
+TWILIO_ACCOUNT_SID=
```

Comment:
These should be `ACCOUNT_SID` and `AUTH_TOKEN` as those two are auto injected by Twilio Functions so no need for them here.